backend/canisters/local_user_index/api/src/queries/c2c_lookup_user_id.rs

```diff
@@ -0,0 +1,14 @@
+use candid::{CandidType, Principal};
```

Comment:
We can probably remove this as it is a subset of the functionality exposed by `c2c_lookup_user`